plans/provision_agent.pp

```diff
@@ -58,7 +58,7 @@
         without_default_logging() || { run_plan(facts, nodes => $target) }
         run_task(
           'sign_cert::sign_cert',
-          $master,
+          $master_fqdn,
```

Comment:
This change breaks the task for me. `$master` is the targetspec, whereas the `$master_fqdn` is not resolvable on my host box.

~~~
==> compiler: {
==> compiler:   "kind": "bolt/run-failure",
==> compiler:   "msg": "Plan aborted: run_task 'sign_cert::sign_cert' failed on 1 nodes",
==> compiler:   "details": {
==> compiler:     "action": "run_task",
==> compiler:     "object": "sign_cert::sign_cert",
==> compiler:     "result_set": [
==> compiler:       {
==> compiler:         "node": "master.puppetdebug.vlan",
==> compiler:         "target": "master.puppetdebug.vlan",
==> compiler:         "action": null,
==> compiler:         "object": null,
==> compiler:         "status": "failure",
==> compiler:         "result": {
==> compiler:           "_error": {
==> compiler:             "kind": "puppetlabs.tasks/connect-error",
==> compiler:             "msg": "Failed to connect to master.puppetdebug.vlan: getaddrinfo: nodename nor servname provided, or not known",
==> compiler:             "details": {
==> compiler:             },
==> compiler:             "issue_code": "CONNECT_ERROR"
==> compiler:           }
==> compiler:         }
==> compiler:       }
==> compiler:     ]
==> compiler:   }
==> compiler: }
~~~

Reply:
Ah, I wasn't sure about that one.  It works for you with just `$master`?

Reply:
Fixed